src/components/Logo/__tests__/Logo.test.js

```diff
@@ -22,30 +24,30 @@ describe('components/Logo', () => {
   });
 
   it('should render correctly', () => {
-    const { container } = render(withAppContext(<Logo />));
+    const { container, getByTestId } = render(withAppContext(<Logo tall />));
 
     expect(container.querySelector(`a[href="${configuration.links.home}"]`)).toBeInTheDocument();
     expect(container.querySelector(`img[src="${configuration.logo.url}"]`)).toBeInTheDocument();
-    expect(container.querySelector(`img[height="${configuration.logo.height}"]`)).toBeInTheDocument();
-    expect(container.querySelector(`img[width="${configuration.logo.width}"]`)).toBeInTheDocument();
+    expect(getByTestId('logo')).toHaveStyleRule('height', configuration.logo.height.toString());
+    expect(getByTestId('logo')).toHaveStyleRule('width', configuration.logo.width.toString());
   });
 
   it('should render the same when tall', () => {
-    const { container } = render(withAppContext(<Logo tall />));
+    const { container, getByTestId } = render(withAppContext(<Logo tall />));
 
     expect(container.querySelector(`a[href="${configuration.links.home}"]`)).toBeInTheDocument();
     expect(container.querySelector(`img[src="${configuration.logo.url}"]`)).toBeInTheDocument();
-    expect(container.querySelector(`img[height="${configuration.logo.height}"]`)).toBeInTheDocument();
-    expect(container.querySelector(`img[width="${configuration.logo.width}"]`)).toBeInTheDocument();
+    expect(getByTestId('logo')).toHaveStyleRule('height', configuration.logo.height.toString());
+    expect(getByTestId('logo')).toHaveStyleRule('width', configuration.logo.width.toString());
   });
```

Comment:
This test case is equal to the previous one. Maybe remove it?

Reply:
Same here, it was there, will remove.